keps/sig-api-machinery/2161-apiserver-default-labels/README.md

```diff
@@ -118,10 +118,9 @@ the namespace's name with traditional label selector mechanisms.
 
 NetworkPolicies allow users to open up traffic to/from their namespaces with the help of `namespaceSelector` field.
 However, there is a concern that arbitrary labels may not be the most secure way to open up traffic when it comes
-to namespace resource. In case a namespace's labels are known by others, any user can add labels at any time to any
-namespace and thus send traffic to a pod belonging to this namespace. Matching a namespace by its name, on the other
-hand, is a more reliable way to whitelist namespaces as it's much easier to specifically allow only namespaces user
-control since user knows their own namespace names.
+to namespace resource. In case a namespace's labels are known by others, any user with wrote access to a namespace can add labels at any time to said namespace, and thus send traffic to a pod belonging to this namespace.
```

Comment:
In the kube default policy, is there any user besides a cluster-admin who can do this?